telegram/forcereply.py

```diff
@@ -31,31 +31,48 @@ class ForceReply(ReplyMarkup):
     to sacrifice privacy mode.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
-    considered equal, if their :attr:`selective` is equal.
+    considered equal, if their :attr:`selective`, and :attr:`input_field_placeholder` are equal.
```

Comment:
strictly speaking, this is breaking and for most objects, we only used the optionals for equality comparison, if it's the "exactly one of the optionals" case … 🤔

Reply:
ooo will revert